diagnostic.md

```diff
@@ -73,13 +88,29 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "POST" "http://www.example.com" \
+--header "Content-Type: application/json" \
+--data $'{
+  "name": "French Cruller",
+  "price": "$0.99"
+}'
+
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function () {
+  let newDonut = {
+    "name": "French Cruller",
+    "price": "$0.99"
+  };
+  return $.ajax({
+    url: http://www.example.com,
+    method: 'POST',
+    data: newDonut
+  });
+};
```

Comment:
Creative way for you to assign newDonut. Again though. Missing object name of `Donut`.